modules/core/src/main/java/org/apache/ignite/internal/processors/rest/handlers/log/GridLogCommandHandler.java

```diff
@@ -161,6 +161,10 @@ private String readLog(int from, int to, File logFile) throws IgniteCheckedExcep
 
                 start++;
             }
+
+            if (start < from || start > (to + 1)){
```

Comment:
I think that you can simply do the following check 

if (content.size() == 0)
thrown new IgniteCheckedException(...)